portfolio/js/models/github.js

```diff
@@ -0,0 +1,44 @@
+(function(module) {
+    // This IIFE holds the logic for what data should end up on the github tab page
+
+    // Constructor function to turn ajax data into gh objects
+    function ghObj(repoData) {
+        for (var key in repoData)
+            this[key] = repoData[key];
+    }
+
+    // Make an array to hold the gh objects returned by ajax
+    ghObj.allRepos = [];
+
+    ghObj.prototype.createRepoHtml = function() {
+        var $sourcehtml = $('#github-template').html();
+        var template = Handlebars.compile($sourcehtml);
+        return template(this);
+    };
+
+    ghObj.requestRepos = function() {
```

Comment:
You could make this more functional by making this function accept a callback parameter, then call the callback at the end of the function.